### PR TITLE
Reader study detail refactor

### PR DESCRIPTION
**Pull request by jmsmkn**
This PR adds a sidebar to the reader study detail page for easier navigation. It is still a single page so likely still slow.

Screenshot_20200224_131521

### CODE REVIEW

File: app/grandchallenge/core/templates/grandchallenge/partials/userlinks.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
             <img class="mugshot"
                  src="{{ user.user_profile.get_mugshot_url }}"
                  alt="User Mugshot"
-                 style="height: 1.5em; vertical-align: middle;"/>
+                 style="height: 1.33em; vertical-align: -25%;"/>
```
Comment (MikeOverkamp-diag) [inline app/grandchallenge/core/templates/grandchallenge/partials/userlinks.html]: Does the -25% work well in all resolutions?

File: app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html [hunk 2]
```diff
@@ -1,24 +1,76 @@
...
+            </div>
+        </div>
+    {% endif %}
+{% endblock %}
 
-    <h1>Reader Study {{ object.title }}</h1>
+{% block content %}
+    {% get_obj_perms request.user for object as "readerstudy_perms" %}
 
     {% if not object.is_valid %}
```
Comment (MikeOverkamp-diag) [inline app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html]: Something I just noticed and might as well be part of this PR: this message also shows for readers. We should also check if the user is an editor.

File: app/grandchallenge/core/templates/grandchallenge/partials/userlinks.html (hunk 1 above)
Comment (jmsmkn) [reply app/grandchallenge/core/templates/grandchallenge/partials/userlinks.html]: I think so, this is to match the alignment from font-awesome.

File: app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html (hunk 2 above)
Comment (jmsmkn) [reply app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html]: Updated

**Pull request merged**